### PR TITLE
Fix spelling mistakes

### PR DESCRIPTION
**Pull request by bengadbois**

### CODE REVIEW

File: dev-tools/vendor/github.com/cavaliercoder/go-rpm/packagefile.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	"time"
 )
 
-// A PackageFile is an RPM package definition loaded directly from the pacakge
+// A PackageFile is an RPM package definition loaded directly from the package
```
Comment (tsg) [inline dev-tools/vendor/github.com/cavaliercoder/go-rpm/packagefile.go]: Can you remove this one please? It's in a dependency so it doesn't help to much to fix the typo in our vendored copy.

File: dev-tools/vendor/github.com/cavaliercoder/go-rpm/packagefile.go [hunk 2]
```diff
@@ -167,7 +167,7 @@ func (c *PackageFile) FileSize() uint64 {
 	return c.fileSize
 }
 
-// Checksum computes and returns the SHA256 checksum (encoded in hexidecimal) of
+// Checksum computes and returns the SHA256 checksum (encoded in hexadecimal) of
```
Comment (tsg) [inline dev-tools/vendor/github.com/cavaliercoder/go-rpm/packagefile.go]: Same on this one.

File: packetbeat/flows/util.go [hunk 3]
```diff
@@ -66,7 +66,7 @@ func (w *worker) tick(t *time.Ticker) bool {
 	}
 }
 
-func (w *worker) periodicaly(tick time.Duration, fn func() error) {
+func (w *worker) periodically(tick time.Duration, fn func() error) {
```
Comment (tsg) [inline packetbeat/flows/util.go]: There's some unit [test failures](https://travis-ci.org/elastic/beats/jobs/193893438#L405) which I suspect are caused by this change.

**Comment by tsg** [comment]
Thanks for the fixes, I left some comments. Out of curiosity, did you use a tool for this? Perhaps we can integrate it in our processes.

**Comment by bengadbois** [comment]
I ran [Gometalinter](https://github.com/alecthomas/gometalinter) which includes a number of opinionated suggestions, but the specific linter that caught these was [misspell](https://github.com/client9/misspell).

[This](https://github.com/mholt/caddy/commit/c7c34266dab44ed222becc4a8eef982cd1707b2a) is how I added it to the Caddy's build.

I've personally not had false positives, but might not want to make that a build passing requirement if you choose to adopt it.

File: packetbeat/flows/util.go (hunk 3 above)
Comment (bengadbois) [inline packetbeat/flows/util.go]: Should be fixed now

**Comment by tsg** [comment]
jenkins, test it

**Comment by tsg** [comment]
Thanks @bengadbois!

**Pull request merged**